### DISCUSSION TITLE
Пароль для пользователя entware в web интерфейсе

### DISCUSSION THREAD

**Discussion by iustinov** [Anonym-tsk/nfqws-keenetic#109] [Q&A] [answered]
Доброго времени суток!

В инструкции в пункте 4 после выполнения команды `opkg install nfqws-keenetic-web` указано 

> Для авторизации введите имя пользователя и пароль пользователя entware

Подскажите пожалуйста какой пароль по умолчанию для этого пользователя&

**Comment by osuaesx** [comment, marked as answer]
login: root
password: keenetic

**Reply by iustinov** [reply]
Спасибо!

**Comment by InfernoDungeon** [comment]
У меня entware был установлен во времена царя гороха, приколюх несколько словил при установке nfqws, а сама веб-морда вообще не запускала дальше ввода пароля, пока не сделал `opkg upgrade`.
